apps/microsoft-teams/frontend/src/components/config/NotificationEditMode/NotificationEditMode.tsx

```diff
@@ -43,8 +47,8 @@ const NotificationEditMode = (props: Props) => {
   };
 
   const handleSave = () => {
-    updateNotification(index, editedNotification);
-    setNotificationIndexToEdit(null);
```

Comment:
Removing this line allows the user to go back to the notification that they're editing